src/level/interface/NPCType.hpp

```diff
@@ -2,11 +2,13 @@
 
 #include <cstdint>
 
+// FIXME: This seems to be a duplicate of "NpcType.hpp" from the GameLoop
```

Comment:
Technically they don't have to overlap - change name to something indicating this is used for level generation